python/hail/expr/aggregators.py

```diff
@@ -724,9 +724,9 @@ def hardy_weinberg(expr) -> StructExpression:
     Returns
     -------
     :class:`.StructExpression`
-        Struct expression with fields `r_expected_het_freq` and `p_hwe`.
+        Struct expression with fields `r_expected_het_freq` and `p_value_hwe`.
     """
-    t = tstruct(r_expected_het_freq=tfloat64, p_hwe=tfloat64)
+    t = tstruct(r_expected_het_freq=tfloat64, p_value_hwe=tfloat64)
```

Comment:
I don't like this change -- it's not consistent with other statistical test methods.

Reply:
I'd also prefer just `p_value`, but hesitated out of wanting consistency with the field name produced by `variant_qc`, and thinking `p_value` would be too vague in that context. But I suppose there is only one p-value produced by variant qc so it's not ambiguous.

I also don't like the `r` in front of `expected_het_freq`. Where we use `r` in sample_qc, its to denote a ratio of the words that follow (e.g. `r_ti_tv`). But `expected_het_freq` is already the right description.

So I propose to change `p_value_hwe` to `p_value` and `r_expected_het_freq` to ``expected_het_freq`, including in variant_qc. Cool?

Reply:
expected het freq _is_ a ratio, no?

Reply:
and I want there to be some hwe identifier for the p value in variantqc

Reply:
yes, but it's NOT the ratio of `expected` to `het_freq` or of `expected_het` to `freq`. It's the ratio of the expected number of hets to the total number of genotypes.

Reply:
In other words, the ratio part is described by `freq`.

Reply:
ok, so I'll make it `p_value_hwe` but only when produced in `variant_qc`. otherwise `p_value`

Reply:
isn't it the ratio of the observed to expected number of hets?

Reply:
and ya that's a good solution re: p value

Reply:
No. That's the point of my rant here:
https://github.com/hail-is/hail/pull/3825

I propose changing it to `het_freq_hwe` everywhere. It's the expected het frequency under HWE.

Reply:
ahhh. OK!